handlers/desired_lrp_handlers.go

```diff
@@ -200,7 +209,11 @@ func (h *DesiredLRPHandler) DesireDesiredLRP(logger lager.Logger, w http.Respons
 
 	err := parseRequest(logger, req, request)
 	if err != nil {
+		logger.Error("failed-parsing-request", err)
 		response.Error = models.ConvertError(err)
+		if err = h.logDesiredLrpParsingErrors(response.Error, request.GetDesiredLrp().GetProcessGuid()); err != nil {
+			logger.Error("failed-logging-parsing-errors", err)
```

Comment:
**Question**

`failed-logging-parsing-errors` is very ambiguous and hard to decipher here. What is the intent of the message? Is it to fail to log when parsing errors? Is it to fail to log parsing errors? Is it to fail `logging parsing errors`?